src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -131,22 +140,26 @@ HARDWARE_INTRINSIC(Sve,           LoadVectorByteNonFaultingZeroExtendToInt64,
 HARDWARE_INTRINSIC(Sve,           LoadVectorByteNonFaultingZeroExtendToUInt16,                      -1,      1,      false, {INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_ldnf1b,     INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_MemoryLoad,            HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           LoadVectorByteNonFaultingZeroExtendToUInt32,                      -1,      1,      false, {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_ldnf1b,     INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_MemoryLoad,            HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           LoadVectorByteNonFaultingZeroExtendToUInt64,                      -1,      1,      false, {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_ldnf1b,     INS_invalid,        INS_invalid},     HW_Category_MemoryLoad,            HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_LowMaskedOperation)
+HARDWARE_INTRINSIC(Sve,           LoadVectorByteZeroExtendFirstFaulting,                            -1,      2,      true,  {INS_invalid,        INS_invalid,        INS_sve_ldff1b,     INS_sve_ldff1b,     INS_sve_ldff1b,     INS_sve_ldff1b,     INS_sve_ldff1b,     INS_sve_ldff1b,     INS_invalid,        INS_invalid},     HW_Category_MemoryLoad,            HW_Flag_Scalable|HW_Flag_ExplicitMaskedOperation|HW_Flag_LowMaskedOperation|HW_Flag_SpecialCodeGen|HW_Flag_SpecialSideEffectMask)
```

Comment:
None of the `LoadVector*FirstFaulting` needs `HW_Flag_SpecialSideEffectMask` because they are already of category `HW_Category_MemoryLoad`.